### ISSUE TITLE
LoadBalancersConfig is missing from spot_fleet_request resource

### ISSUE THREAD

**Issue by davidneudorfer** [terraform-providers/terraform-provider-aws#3633]
### Terraform Version
Terraform v0.11.3
AWS Provider v1.9.0

### Affected Resource(s)
Please list the resources as a list, for example:
- spot_fleet_request

### Expected Behavior
"LoadBalancersConfig" is missing from the spot_fleet_request resource.

### Important Factoids
AWS API includes LoadBalancersConfig in `--spot-fleet-request-config` JSON
https://docs.aws.amazon.com/cli/latest/reference/ec2/request-spot-fleet.html

```
  "LoadBalancersConfig": {
    "ClassicLoadBalancersConfig": {
      "ClassicLoadBalancers": [
        {
          "Name": "string"
        }
        ...
      ]
    },
```

But its missing from the terraform [module](https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/resource_aws_spot_fleet_request.go) and [documentation](https://www.terraform.io/docs/providers/aws/r/spot_fleet_request.html).

**Comment by davidneudorfer** [comment]
@bflad this is fixed by https://github.com/terraform-providers/terraform-provider-aws/pull/2564

**Comment by bflad** [comment]
Fantastic news! Thanks for pointing me at that open PR. Taking a look now as it looks to be in pretty good shape at first glance. 😄

**Comment by bflad** [comment]
Support has been merged into master and will be released in v1.11.0 of the AWS provider, likely at the end of the week.

**Comment by bflad** [comment]
This has been released in [version 1.11.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1110-march-09-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
